ospd/ospd.py

```diff
@@ -1543,11 +1550,14 @@ def get_vts_xml(self, vt_id=None, filtered_vts=None):
 
         vts_xml = Element('vts')
 
-        if vt_id:
-            vts_xml.append(self.get_vt_xml(vt_id))
-        elif filtered_vts:
+        if filtered_vts is not None and len(filtered_vts) == 0:
```

Comment:
Could possibly be simplified as
```suggestion
        if filtered_vts is not None and not filtered_vts:
```
but i am not sure if that is more readable then before.

Reply:
I would prefer to leave it as it is, with an explicit "== 0", since your suggestion is true also for None and False.

Reply:
I am fine with leaving it as it is